### PR TITLE
WSAS-2129: Port offset change to avoid build failures

### PR DESCRIPTION
**Pull request by Manuri**

### CODE REVIEW

File: modules/integration/tests-common/common-artifacts/jaxrs/jaxrs-client-test-webapp/src/main/java/org/wso2/as/integration/JAXRSCXFClientTestServlet.java [hunk 1]
```diff
@@ -50,6 +50,12 @@ public JAXRSCXFClientTestServlet() {
      */
     protected void doGet(HttpServletRequest request, HttpServletResponse response)
             throws ServletException, IOException {
+        String hostIp = request.getParameter("Host");
+        String port = request.getParameter("Port");
+        if (hostIp != null && port != null) {
+            host = "http://" + hostIp + ":" +port + "/";
```
Comment (callkalpa) [inline modules/integration/tests-common/common-artifacts/jaxrs/jaxrs-client-test-webapp/src/main/java/org/wso2/as/integration/JAXRSCXFClientTestServlet.java]: Formatting issue

File: modules/integration/tests-common/common-artifacts/jaxrs/jaxrs-client-test-webapp/src/main/java/org/wso2/as/integration/JAXRSClientTestServlet.java [hunk 2]
```diff
@@ -55,6 +55,13 @@ public JAXRSClientTestServlet() {
      */
     protected void doGet(HttpServletRequest request, HttpServletResponse response)
             throws ServletException, IOException {
+        String hostIp = request.getParameter("Host");
+        String port = request.getParameter("Port");
+        if (hostIp != null && port != null) {
+            host = "http://" + hostIp + ":" +port + "/";
```
Comment (callkalpa) [inline modules/integration/tests-common/common-artifacts/jaxrs/jaxrs-client-test-webapp/src/main/java/org/wso2/as/integration/JAXRSClientTestServlet.java]: Formatting issue

File: modules/integration/tests-integration/tests/src/test/java/org/wso2/appserver/integration/tests/webapp/mgt/CARBON15199CarbonContextAppNameNullTest.java [hunk 3]
```diff
@@ -105,8 +105,8 @@ public void configureServer() throws AutomationFrameworkException {
                     userPopulator.populateUsers();
                     testServerManager.stopServer();
                     carbonHome = testServerManager.getCarbonHome();
-//                    String commandDirectory = carbonHome + File.separator + "bin";
-//                    String[] cmdArray;
+                    //                    String commandDirectory = carbonHome + File.separator + "bin";
+                    //                    String[] cmdArray;
```
Comment (callkalpa) [inline modules/integration/tests-integration/tests/src/test/java/org/wso2/appserver/integration/tests/webapp/mgt/CARBON15199CarbonContextAppNameNullTest.java]: Is this necessary?

File: modules/integration/tests-integration/tests/src/test/resources/automation.xml [hunk 4]
```diff
@@ -191,40 +191,40 @@
...
+            <!--<user key="userm1">-->
+            <!--<userName>testuse11</userName>-->
+            <!--<password>testuse11</password>-->
+            <!--</user>-->
+            <!--<user key="userm2">-->
+            <!--<userName>testuse21</userName>-->
+            <!--<password>testuse21</password>-->
+            <!--</user>-->
+            <!--</users>-->
             <!--</tenant>-->
```
Comment (callkalpa) [inline modules/integration/tests-integration/tests/src/test/resources/automation.xml]: Seems there are no real change, shall we keep them as they were, without formatting?

File: modules/integration/tests-integration/tests/src/test/resources/automation.xml [hunk 5]
```diff
@@ -302,16 +306,16 @@
...
                 <class>
                     <name>org.wso2.carbon.integration.common.extensions.usermgt.UserPopulateExtension</name>
                 </class>
                 <!--<class>-->
-                    <!--<name>org.wso2.carbon.automation.extensions.servers.coverageServer.RemoteCoverageServerExtension</name>-->
-                    <!--<parameter name="coverageServerPort" value="6331" />-->
-                    <!--<parameter name="coverageDumpFilePath" value="/Users/krishantha/Desktop/jacoco" />-->
+                <!--<name>org.wso2.carbon.automation.extensions.servers.coverageServer.RemoteCoverageServerExtension</name>-->
+                <!--<parameter name="coverageServerPort" value="6331" />-->
+                <!--<parameter name="coverageDumpFilePath" value="/Users/krishantha/Desktop/jacoco" />-->
```
Comment (callkalpa) [inline modules/integration/tests-integration/tests/src/test/resources/automation.xml]: Seems this addition is not necessary

File: modules/integration/tests-integration/tests/src/test/java/org/wso2/appserver/integration/tests/carbontools/TenantIdleTimeCommandTestCase.java [hunk 6]
```diff
@@ -59,20 +72,20 @@ public void init() throws Exception {
     @Test(groups = "wso2.as", description = "Clean tenants after idle time")
     public void testTenantIdleTime() throws Exception {
 
-        AutomationContext context =
+       /* AutomationContext context =
```
Comment (thusithathilina) [inline modules/integration/tests-integration/tests/src/test/java/org/wso2/appserver/integration/tests/carbontools/TenantIdleTimeCommandTestCase.java]: can we remove the commented blocks?

Comment (Manuri) [inline modules/integration/tests-integration/tests/src/test/java/org/wso2/appserver/integration/tests/carbontools/TenantIdleTimeCommandTestCase.java]: Corrected in https://github.com/wso2/product-as/pull/346

File: modules/integration/tests-integration/tests/src/test/java/org/wso2/appserver/integration/tests/server/mgt/OSGIServerBundleStatusTestCase.java [hunk 7]
```diff
@@ -18,8 +18,30 @@
...
+    private HashMap<String, String> serverPropertyMap = new HashMap();
+    private MultipleServersManager manager = new MultipleServersManager();
+    private static int telnetPort = 2000;
+
+    @BeforeClass(alwaysRun = true)
+    public void init() throws XPathExpressionException, AutomationFrameworkException {
+        this.serverPropertyMap.put("-DportOffset", "101");
+        this.serverPropertyMap.put("-DosgiConsole", Integer.toString(telnetPort));
+        AutomationContext autoCtx = new AutomationContext();
+        CarbonTestServerManager server = new CarbonTestServerManager(autoCtx, System.getProperty("carbon.zip"), this.serverPropertyMap);
```
Comment (Nipuni) [inline modules/integration/tests-integration/tests/src/test/java/org/wso2/appserver/integration/tests/server/mgt/OSGIServerBundleStatusTestCase.java]: Please fix formatting issue, line exceeds 120 characters.

Comment (Manuri) [inline modules/integration/tests-integration/tests/src/test/java/org/wso2/appserver/integration/tests/server/mgt/OSGIServerBundleStatusTestCase.java]: Fixed in https://github.com/wso2/product-as/pull/346

**Pull request merged**